### DISCUSSION TITLE
EMBL Grid vs left positioned content pages

### DISCUSSION THREAD

**Discussion by khawkins98** [visual-framework/vf-core#1506] [Proposals]
A recurring bit of feedback and an "itch" is that the `embl-grid` feels not-quite-right when used with the `vf-hero` and simple page content — example:

---

![image](https://user-images.githubusercontent.com/928100/116224738-4b277a00-a751-11eb-99da-e6c6ca0cde8a.png)

---

These are currently often manually overridden like `vf-grid vf-grid__col-3 > vf-grid__col--span-2`:

---

![image](https://user-images.githubusercontent.com/928100/116224714-42cf3f00-a751-11eb-831a-8c14a22d631a.png)

---

Functions like:

![image](https://user-images.githubusercontent.com/928100/116226357-2a602400-a753-11eb-962d-6f926e6d9572.png)

---

This current situation presents a number of issues:

- The workaround is a bit of a hack that is cumbersome to do and more subject to accidental breakdown
- and, more impotently: it creates inconsistency across EMBL pages and confusion around the correct way to make pages 

## Note

The `embl-grid` is still of use on "inside" pages where there is a well thought-out architecture; examples:

![image](https://user-images.githubusercontent.com/928100/116224852-6befcf80-a751-11eb-82f0-8e156f703885.png)

![image](https://user-images.githubusercontent.com/928100/116224906-7b6f1880-a751-11eb-82fc-5fe38b273274.png)

![image](https://user-images.githubusercontent.com/928100/116224933-84f88080-a751-11eb-8338-b4526f80f3d3.png)

## So...

How can we address the visual issue and support the use case when the `vf-hero` is used on a page with simple content?

After discussing with the Digital Team, here's the current thinking.

1. We standardise the "left layout" by making a “vf-content--layout” that has the 2-over-3 columns with a sidebar (3rd column)
2. `vf-content--layout` is _only_ recommended for use with:
- “tail end” pages that are at the end of a user’s journey, _and_ have a `vf-hero`
- do **not** use on landing pages/gateway pages to other content (Gateway pages should have a more thought out content architecture and be able to make use of the `embl-grid`)

We're interested to know if this also seems like a workable approach for other users.

**Comment by khawkins98** [comment]
This "unofficial" grid is also coded into the top of every group microsite landing page:

![image](https://user-images.githubusercontent.com/928100/116563116-04c64c80-a904-11eb-961e-5b78474818a0.png)

**Comment by filipe-dias** [comment]
Not much traction on the subject so far :(
